src/Plugin.hs

```diff
@@ -90,13 +90,11 @@ questionToMeta b = withBufferStuff b $ \bs -> do
     getGoalContents_maybe b int >>= \case
       -- We only don't have a goal contents if we are a ? goal
       Nothing -> do
-        replaceInterval b (iStart int) (iEnd int) "{! !}"
+        replaceInterval b int "{! !}"
         let int' = int
-                  { iEnd = (iStart int)
-                              -- Inclusive, so we add only 4 offset, rather
-                              -- than the 5 for the characters
-                    { p_col = offsetPlus (p_col $ iStart int) (Offset 4)
-                    }
+                  { iEnd = (iStart int) `addCol` Offset 4
+                    -- Inclusive, so we add only 4 offset, rather
```

Comment:
Mind updating this comment to say that *highlight intervals* are  inclusive? Maybe this is another thing we should track at the type level.

Reply:
The 1 that is removed here was added back in the `ec + 1` line.

- highlighting intervals are actually exclusive (see my other comment above)
- since I removed the `ec + 1` this `Offset 4` is off-by-one, this should be `Offset 5` now.
-  When you do a `QuestionToMeta` Agda actually gets reloaded so its highlighting clobbers this one. I just confirmed that indeed this sends an interval which is 1 byte shorter than the interval Agda sends us right afterwards so we don't see it.